src/main/java/make_comment/interface_adapter/makeCommentController.java

```diff
@@ -0,0 +1,24 @@
+package make_comment.interface_adapter;
+
+import make_comment.use_case.MakeCommentInputBoundary;
+import make_comment.use_case.MakeCommentRequestModel;
+
+public class makeCommentController {
+    private final MakeCommentInputBoundary interactor;
+
+    public makeCommentController(MakeCommentInputBoundary interactor) {
+        this.interactor = interactor;
+    }
+
+    public void passToInteractor(String body, int postId){
+        int commentId = interactor.getNumCommentCreated();
+        int currentUserID = interactor.getCurrentUserID();
+        commentId ++;
```

Comment:
Increasing the Id should also be done in the interactor I think since only the interactor manipulates the entities.